modules/installation-user-infra-machines-advanced-console-configuration.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installation-user-infra-machines-advanced-console-configuration_{context}"]
+= Default console configuration
+
+{op-system-first} nodes installed from an {product-title} {product-version} boot image use a default console that is meant to accomodate most virtualized and bare metal setups. Different cloud and virtualization platforms may use different defaults depending on the chosen architecture. Bare metal installations use the kernel defaults which typically means the graphical console is the primary console and the serial console is disabled.
+
+The default consoles may not match your specific hardware configuration or you might have specific needs that require you to adjust the default console. For example:
+
+* You want to access the emergency shell on the console for debugging purposes.
+* Your cloud platform does not provide interactive access to the graphical console, but provides a serial console.
+* You want to enable multiple consoles.
+
+Console configuration is inherited from the boot image. This means that new nodes in existing clusters are unaffected by changes to the default console.
+
+You can configure the console for bare metal installations in a couple of ways:
+
+* Using `coreos-installer` manually on the command line.
+* Using the `coreos-installer iso` or `coreos-installer pxe` subcommands with the `--dest-console` option to create a custom image that automates the process.
+//* During provisioning of a cloud or virtual machine, using Ignition.
+//* Using `rpm-ostree`, after installation.
```

Comment:
Did you mean to uncomment those?

Reply:
I commented them out because I wasn't sure they were for bare metal installs, though I would like to reuse this concept module for other install docs